### DISCUSSION TITLE
Publisher IP settings

### DISCUSSION THREAD

**Discussion by markos2000** [ant-media/Ant-Media-Server#5439] [Q&A]
Hi there, I followed the documentation on how to add publisher IP so I can limit who can stream to the server but I still can stream to the server from different IP addresses even after I specified one IP address only.
example: **_allowedPublisherCIDR=5.161.145.227_**

Thanks!

**Comment by USAMAWIZARD** [comment]
Hi @markos2000 ,
Which protocol are you using for publishing the stream ? allowedPublisherCIDR only works for RTMP publishing.
Sorry it was not mentioned in the docs I will update the docs.

**Reply by markos2000** [reply]
Hi, I'm currently using RTMP to publish the stream through obs.

**Reply by USAMAWIZARD** [reply]
Hi , I  tried on 2.6.1 and its working fine for me , please check if you are entering correct public ip https://whatismyipaddress.com/

**Reply by markos2000** [reply]
Hi, I'm currently using Community Edition 2.6.2 20230623_0842. if I have the wrong IP address in the settings then I shouldn't be able to connect from any IP address right? I tried multiple devices with different IP addresses on each, and I can still publish to the server no matter what IP address I put in the settings, even if I put 127.0.0.1 as the filtred IP and the server is not locally hosted. I attached a screenshot from the settings.

**Reply by USAMAWIZARD** [reply]
attached image not visible

**Reply by markos2000** [reply]
Hi, sorry about the image not sure why it didn't load but here's my full settings file for the LiveApp application. Thanks!
https://www.codebin.cc/code/cll69w52a0001ld087q289pom:Day8PR95naCKVsnbeLTeVfm6HnaZQu6KgUDqyzoHJdT4

**Reply by USAMAWIZARD** [reply]
have you restarted the server after configuration ?
`sudo systemctl restart antmedia `

in red5 file it should look something like this 
settings.allowedPublisherCIDR=103.171.201.109

**Reply by markos2000** [reply]
Hmm... I don't have it like that in the red5 file, let me modify it and recheck.

**Reply by USAMAWIZARD** [reply]
red5-web.properties file, which is located in the directory path that looks like this: 

/usr/local/antmedia/YOUR_APP_NAME/WEB_INF/red5-web.properties

settings.allowedPublisherCIDR=IP_ADDRESS

save the file 

restart the server `sudo systemctl restart antmedia `

**Reply by markos2000** [reply]
I replaced `allowedPublisherCIDR=5.161.145.227` with `settings.allowedPublisherCIDR=5.161.145.227` in the red5-web.properties (**_antmedia/webapps/LiveApp/WEB-INF_**) and did `sudo systemctl restart antmedia` and still can connect. is there an IP database maybe old IP addresses are saved there? is the reason that I'm still able to connect?

**Reply by USAMAWIZARD** [reply]
Can you please book a free consultation  meeting  [here](https://antmedia.io/) 

![image](https://github.com/ant-media/Ant-Media-Server/assets/47350008/869b0931-bf81-4979-bce3-6610e9991f60)
